install/kubernetes/helm/istio/templates/sidecar-injector-configmap.yaml

```diff
@@ -54,10 +54,13 @@ data:
         imagePullPolicy: {{ .Values.global.imagePullPolicy }}
         securityContext:
           capabilities:
+            drop:
+            - "all"
             add:
-            - NET_ADMIN
-          privileged: true
-        restartPolicy: Always
+            - "NET_ADMIN"
+            - "NET_RAW"
+          privileged: false
+          allowPrivilegeEscalation: false
```

Comment:
Why drop the restartPolicy ?

Same comment - stricter security is great, but I don't think this has been tested on enough platforms - so it's better to have it as an option, and after we confirm it works we can enable as default.

Reply:
I've added back missing `restartPolicy: Always`, it was removed by mistake.